code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -100,6 +100,79 @@
 	item_state = "ihscombat"
 	armor_list = list(melee = 10, bullet = 60, energy = 10, bomb = 10, bio = 0, rad = 0)
 
+/obj/item/clothing/gloves/thick/swat/blackshield_corpsman
+	name = "corpsman combat gloves"
+	desc = "A pair of fire and impact-resistant security combat gloves. This pare has an extra rubbery coating to help prevent germs."
```

Comment:
```suggestion
	desc = "A pair of fire and impact-resistant security combat gloves. This pair has an extra rubbery coating to help prevent germs."
```

I also suggest adding `permeability_coefficient = 0.01` and `germ_level = 0` vars so that it's on par with both its description stating that it helps prevent germs (and chem spills), and latex gloves.